### DISCUSSION TITLE
Fooocus running flawlessly on GTX 1050ti with 4gb gpu memory

### DISCUSSION THREAD

**Discussion by harold1110** [lllyasviel/Fooocus#1994] [General]
![Screenshot 2024-01-20 124200](https://github.com/lllyasviel/Fooocus/assets/145927154/b6dbc08d-98f6-4bf1-a2d6-b0b0d333fe1a)
Hey there everyone and developers of Fooocus.

So I read the minimum requirements to run Fooocus and I saw that my gpu might now support running it, Still I tried do so and it actually worked with no errors or what so ever. It takes me around 5-6 mins to generate 1 image which is quite acceptable considering my specs. I just wanted to let you all know bout it so it might be of some help to few of you out there

Here is the list of my components
CPU - Intel(R) Core(TM) i3-6098P CPU @ 3.60GHz   3.60 GHz
GPU - Zotac Nvidia Gefore GTX 1050ti 4gb Graphic Memory
Ram - Corsair Vengeance LPX 16GB (1x16GB) DDR4 3200MHZ (but is limited 2200mhz due to my motherboard capability)
OS - Windows 10 Pro 22H2

That's all!
Have a great day!
Cheers!

**Comment by Irmagon** [comment]
Pls try this presets https://github.com/lllyasviel/Fooocus/discussions/1888
I wery inreresting how it works on 1050

**Reply by MAZASA-mzs** [reply]
It seems I'm a little late with my personal tests, but: it works, and works well (1-2 minutes, or even faster with the default settings: Extreme Speed)

UPD:
I used `Segmind-VegaRT a distilled consistency adapter for Segmind-Vega` model
`The Segmind-Vega Model is a compact version with a remarkable 70% reduction in size compared to the Base SDXL Model.` is rather slow.

Outputs from light model:

<img width="1024" height="1024" alt="image" src="https://github.com/user-attachments/assets/a255fea8-97bb-4e2d-be6e-cc7689f8c350" />

And for prompt: "rat"
<img width="1024" height="1024" alt="image" src="https://github.com/user-attachments/assets/350f30ca-51fc-4b24-b16a-5d5ad1996228" />
